smart_card_connector_app/src/background.js

```diff
@@ -86,7 +86,8 @@ function launchedListener() {
 }
 
 function openWindow() {
-  GSC.PopupWindow.Server.createWindow(WINDOW_URL, WINDOW_OPTIONS);
+  GSC.PopupWindow.Server.createWindow(WINDOW_URL, WINDOW_OPTIONS,
+      messageChannelPool.addOnUpdateListener.bind(messageChannelPool));
```

Comment:
It would be a bit more readable if you wrap this into some dictionary, e.g.:
`{'clientAppListUpdateSubscriber': messageChannelPool.addOnUpdateListener.bind(messageChannelPool)}`